ocs_ci/ocs/ocp.py

```diff
@@ -506,3 +509,41 @@ def switch_to_default_rook_cluster_project():
         bool: True on success, False otherwise
     """
     return switch_to_project(defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def rsync(src, dst, node, dst_node=True, extra_params=""):
```

Comment:
I wanted to add this last week but missed, regardless.

function signature could have been better, instead of `src` it should be `src_path` and `dst` it should be `dst_path` to make it clear between path and nodes.

`node` -> `src_node` ( should maintain consistency)
and `extra_params=None`